### PR TITLE
revamp station to directory

### PR DESCRIPTION
**Pull request by worlddlckgh**

### CODE REVIEW

**Review by benlvb — APPROVED** [review]

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,177 +1,570 @@
...
-    tweet: `I have so much admiration for @weremeow and the @JupiterExchange team 🙏
-    
-    Truly Solana aligned and leading the way in transparent and honest communication.
-    
-    Not to mention that LFG Launchpad actually worked super well and will kick-start a new era of fair token launches`,
-    time: new Date(2024, 1, 1, 14, 5),
-    permalink: "https://twitter.com/KultureElectric/status/1752936182785130819",
-  },
-];
+        <a
```
Comment (wkitsin) [inline src/pages/index.tsx:33]: i think we should use `target=_blank` for this, so it opens another tab.

**Review by wkitsin — COMMENTED** [review]
CleanShot 2024-03-05 at 5  13 06


is the lack of padding on mobile intentional ?

**Comment by worlddlckgh** [comment]
> CleanShot 2024-03-05 at 5  13 06
> 
> is the lack of padding on mobile intentional ?

intentional, "Launch Limit Order" too long

File: src/pages/index.tsx (hunk 1 above)
Comment (worlddlckgh) [reply src/pages/index.tsx:34]: this is done, along with ref rel

Comment (worlddlckgh) [reply src/pages/index.tsx:34]: this is done, along with ref rel

**Review by wkitsin — APPROVED** [review]

**Pull request merged**